pkg/util/version/const.go

```diff
@@ -64,7 +64,7 @@ var (
 
 // FluentbitImage contains the location of the Fluentbit container image
 func FluentbitImage(acrDomain string) string {
-	return acrDomain + "/fluentbit:1.7.8-1"
+	return acrDomain + "/fluentbit:1.9.9-1"
```

Comment:
Should we try to add some way to populate this fluentbit version dynamically? It updates pretty often and the older versions aren't typically available anymore as fluentbit releases. Otherwise this change looks great and the impact is measurable.

Reply:
This compiles them, so the requirement for an upstream release is not required.